src/PackageVersions/Installer.php

```diff
@@ -77,6 +77,13 @@ public static function dumpVersionsClass(Event $composerEvent)
     {
         $io = $composerEvent->getIO();
 
+        if (substr(__DIR__, 0, 7) === 'phar://') {
```

Comment:
It's a hack and the only way I could fix it. I haven't found anyway to get the package root in travis, installed as a package (root/vendor/ocramius/package-version) and from a cloned repo at the same time.

I don't think it breaks the package since it only protects itself from writing Version.php inside the composer.phar file. The question here is if it is a composer bug or not. I don't think a plugin shouldn't get into this situation.

Maybe place a warning in the readme and leave it as it is now.